web/src/pages/onboarding.tsx

```diff
@@ -329,12 +329,11 @@ const OnboardingPage = (props: Props): ReactElement => {
 
         if (newProfileData.operatingPhase === "GUEST_MODE_OWNING") {
           updateQueue.queuePreferences({
-            visibleSidebarCards: [
-              ...updateQueue.currentBusiness().preferences.visibleSidebarCards.filter((cardId: string) => {
-                return cardId !== "welcome";
+            visibleSidebarCards: updateQueue
+              .currentBusiness()
+              .preferences.visibleSidebarCards.filter((cardId: string) => {
+                return cardId !== "welcome" && cardId !== "welcome-up-and-running";
```

Comment:
[pebble] This whole section should be removed, right? The whole thing it was doing is replacing "welcome" with "welcome-up-and-running"...neither of which exist anymore